app/types/tsconfig.json

```diff
@@ -1,7 +1,9 @@
 {
   "compilerOptions": {
     "strict": true,
-    "lib": ["es2015", "dom"]
+    "lib": ["es2015", "dom"],
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true
```

Comment:
```suggestion
```
This option is enabled but I don't see any change that utilizes it. Also, it is true by default if `esModuleInterop` is true anyways. So, I think we can get rid of this line to not pollute the config.

Reply:
You're right, I was using this before adding `import type * as` into proxy files and I was trying TS to stop complaining, gonna remove this change